### PR TITLE
Blog: move "refs to custom components..." fixes #4305

### LINKED ISSUE #4305: v0.14 Refs no longer components
https://facebook.github.io/react/blog/2015/07/03/react-v0.14-beta-1.html
> The other big change we're making in this release is exposing refs to DOM components as the DOM node itself. That means: we looked at what you can do with a ref to a DOM component and realized that the only useful thing you can do with it is call this.refs.giraffe.getDOMNode() to get the underlying DOM node.

I’ve used `refs` to perform a custom method such as `play()` on a child component. Having only the DOM node accessible will make this no longer possible.

Is there a better practice I should be following to call methods on descendent components?

### PR DESCRIPTION
**Pull request by brigand**
Just to prevent the TL;DR confusion. Separate paragraph and moved up above the code block. `s/References/Refs` for consistency.

### CODE REVIEW

**Comment by brigand** [comment]
cc @jimfb

File: docs/_posts/2015-07-03-react-v0.14-beta-1.md [hunk 1]
```diff
@@ -66,7 +68,7 @@ var Zoo = React.createClass({
 });
 ```
 
-This change also applies to the return result of `ReactDOM.render` when passing a DOM node as the top component. References to custom component classes work exactly as before.
+This change also applies to the return result of `ReactDOM.render` when passing a DOM node as the top component.
```
Comment (jimfb) [inline docs/_posts/2015-07-03-react-v0.14-beta-1.md]: Maybe also put some text here to drive the point home?

Some thing like: This change also applies to the return result of `ReactDOM.render` when passing a DOM node (eg. `span`, `div`, `img`) as the top component.  As with refs, this change does not affect custom components (eg. `<MyFancyMenu>` or `<MyContextProvider>`), which remain unaffected by this change.

**Comment by jimfb** [comment]
cc @spicyj There has already been a few people who have been confused by the old wording, as per https://github.com/facebook/react/issues/4305.

File: docs/_posts/2015-07-03-react-v0.14-beta-1.md (hunk 1 above)
Comment (brigand) [inline docs/_posts/2015-07-03-react-v0.14-beta-1.md]: Fixed, thanks.

**Comment by spicyj** [comment]
Sure, thanks.

**Pull request merged**